### PR TITLE
PM-561 BPAY JUnit tests

### PR DESCRIPTION
**Pull request by andrearovere**

### CODE REVIEW

File: src/main/java/it/pagopa/pm/gateway/client/util/Constants.java [hunk 1]
```diff
@@ -15,6 +15,6 @@
 
     public static final String TOKEN =System.getProperty("bancomatPay.client.token");
```
Comment (infantesimone) [inline src/main/java/it/pagopa/pm/gateway/client/util/Constants.java:17]: we can use Value spring annotation to get properties? https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/annotation/Value.html

File: src/main/java/it/pagopa/pm/gateway/Application.java [hunk 2]
```diff
@@ -11,17 +12,17 @@
 @ComponentScan(basePackages = "it.pagopa.pm.gateway")
 @EnableAutoConfiguration(exclude = DataSourceAutoConfiguration.class)
 @Configuration
-public class PaymentTransactionsGatewayApplication extends SpringBootServletInitializer {
+public class Application extends SpringBootServletInitializer {
```
Comment (infantesimone) [inline src/main/java/it/pagopa/pm/gateway/Application.java:15]: why is this renaming needed?

File: src/main/java/it/pagopa/pm/gateway/client/util/ClientConfig.java [hunk 3]
```diff
@@ -1,13 +1,12 @@
 package it.pagopa.pm.gateway.client.util;
 
-import it.pagopa.pm.gateway.client.payment.gateway.client.BancomatPayClientV2;
-import it.pagopa.pm.gateway.client.restapiCD.RestapiCdClient;
+import it.pagopa.pm.gateway.client.bpay.*;
```
Comment (infantesimone) [inline src/main/java/it/pagopa/pm/gateway/client/util/ClientConfig.java:3]: there are differing opinions on using ( * ) for java import, but in this case, what can be the advantage?

**Review by d-bert — APPROVED** [review]

File: src/main/java/it/pagopa/pm/gateway/client/util/Constants.java (hunk 1 above)
Comment (d-bert) [reply src/main/java/it/pagopa/pm/gateway/client/util/Constants.java:17]: Will be changed in next commit

**Pull request merged**